<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win_second</class>
 <widget class="QDialog" name="win_second">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <widget class="QPushButton" name="button_logout">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>119</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#button_logout {
   background-image: url(:/button/button_login.png);
     border-style: inset;
     border-width: 0px;
     border-radius:0px;
     border-top:0px;	
	 border-bottom:0px;
     
    /*font-size:11pt;*/
    font-weight: bold;
    color: #FFFFFF;
 }
 QPushButton#button_logout:pressed {
   background-image: url(:/button/button_login_pres.png);
 }</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>310</y>
     <width>119</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#button_add {
   background-image: url(:/button/button_login.png);
     border-style: inset;
     border-width: 0px;
     border-radius:0px;
     border-top:0px;	
	 border-bottom:0px;
     
    /*font-size:11pt;*/
    font-weight: bold;
    color: #FFFFFF;
 }
 QPushButton#button_add:pressed {
   background-image: url(:/button/button_login_pres.png);
 }</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_name_tab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>670</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
 background-color: #FCFCFC;
border: 1px solid  #d9d9d9;

    color: #868686;
 }

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>91</y>
     <width>668</width>
     <height>200</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget
{
	border: 0px outset  #d9d9d9;

}

QListWidget::Item{
padding-top:9px;
 padding-bottom:9px;
  border: 0px outset  #d9d9d9;
margin:0px -1px 0 0;
 }

QListWidget::Item:text { color: #888888; }


QScrollBar:vertical {

  border-color:#efefef;
  border-width: 3px;
  border-style: solid;

  background-color:#fafafa;
  width: 15px;
  margin:-2px -2px -2px -2px;
}

QScrollBar::handle:vertical {/*бегунок*/

	background-color: #c1c1c1;
	min-height: 30px;
	border-radius:4px;
border: 1px solid #d9d9d9;
 /* 	border-color: #ffffff;
  border-style: solid;*/
/*border-image: url(:/res/scroll/scroll_holder.png) 0% center no-repeat;*/
/*border-radius: 3px;*/
/*image: url(:/res/scroll/scroll_holder.png);*/
 margin:3 2px 3 2px;

}


QScrollBar::sub-line:vertical {/* верхняя стрелка*/
   height: 0;
}

QScrollBar::add-line:vertical {/*нижняя стрелка*/
    height: 0;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
/*при нажатии на пустое место скрола*/
   background: none;
}

</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>670</width>
     <height>202</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
 background-color: #FFFFFF;
border: 1px solid  #d9d9d9;

 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>button_logout</zorder>
  <zorder>button_add</zorder>
  <zorder>label_name_tab</zorder>
  <zorder>table</zorder>
 </widget>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
